_includes/javascripts.html

```diff
@@ -1,9 +1,13 @@
 {% unless site.local_fonts %}
-  <script src="https://ajax.googleapis.com/ajax/libs/webfont/1.6.16/webfont.js"></script>
+  <script src="https://cdnjs.cloudflare.com/ajax/libs/webfont/1.6.27/webfontloader.js"></script>
```

Comment:
Google Analytics is different because the file is updated by Google periodically, this one is a specific version. Isn't this available on bower anyway?